src/input.ts

```diff
@@ -32,7 +41,19 @@ export class Input {
     this.repo = validatedParams.repo
     this.packageName = validatedParams.packageName
     this.numOldVersionsToDelete = validatedParams.numOldVersionsToDelete
+    this.minVersionsToKeep = validatedParams.minVersionsToKeep
+    this.ignoreVersions = validatedParams.ignoreVersions
+    this.deletePreReleaseVersions = validatedParams.deletePreReleaseVersions
     this.token = validatedParams.token
+
+    if (this.minVersionsToKeep > 0) {
+      this.numOldVersionsToDelete = 100 - this.minVersionsToKeep
```

Comment:
do we set limit to 100 if just `num-old-versions-to-delete` is provided and `min-versions-to-keep:` and `  delete-only-pre-release-versions:` are not provided?

Reply:
The limit of 100 is for graphql api so we are adjusting the package versions to be queried around this. So we always maintain `num-old-versions-to-delete`+`min-versions-to-keep` = 100.